tests/acceptance/expected-failures-webUI-on-OCIS-storage.md

```diff
@@ -56,33 +56,11 @@ Other free text and markdown formatting can be used elsewhere in the document if
-- [webUISharingAutocompletion/shareAutocompletionSpecialChars.feature:56](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAutocompletion/shareAutocompletionSpecialChars.feature#L56)
-- [webUISharingAutocompletion/shareAutocompletionSpecialChars.feature:73](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAutocompletion/shareAutocompletionSpecialChars.feature#L73)
-- [webUISharingAutocompletion/shareAutocompletionSpecialChars.feature:74](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAutocompletion/shareAutocompletionSpecialChars.feature#L74)
-- [webUISharingAutocompletion/shareAutocompletionSpecialChars.feature:75](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAutocompletion/shareAutocompletionSpecialChars.feature#L75)
-
-### [Minimum of three characters required for autocompletion](https://github.com/owncloud/ocis/issues/1961)
-- [webUISharingAutocompletion/shareAutocompletion.feature:32](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAutocompletion/shareAutocompletion.feature#L32)
-- [webUISharingAutocompletion/shareAutocompletion.feature:44](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAutocompletion/shareAutocompletion.feature#L44)
-- [webUISharingAutocompletion/shareAutocompletion.feature:87](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAutocompletion/shareAutocompletion.feature#L87)
-- [webUISharingAutocompletion/shareAutocompletion.feature:101](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAutocompletion/shareAutocompletion.feature#L101)
-
 ### [Cannot create users with special characters](https://github.com/owncloud/ocis/issues/1417)
-- [webUISharingAutocompletion/shareAutocompletionSpecialChars.feature:36](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAutocompletion/shareAutocompletionSpecialChars.feature#L36)
 - [webUISharingAutocompletion/shareAutocompletionSpecialChars.feature:37](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAutocompletion/shareAutocompletionSpecialChars.feature#L37)
 - [webUISharingAutocompletion/shareAutocompletionSpecialChars.feature:38](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAutocompletion/shareAutocompletionSpecialChars.feature#L38)
 - [webUISharingAutocompletion/shareAutocompletionSpecialChars.feature:39](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAutocompletion/shareAutocompletionSpecialChars.feature#L39)
+-   [webUISharingAutocompletion/shareAutocompletionSpecialChars.feature:40](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAutocompletion/shareAutocompletionSpecialChars.feature#L40)
```

Comment:
```suggestion
-  [webUISharingAutocompletion/shareAutocompletionSpecialChars.feature:40](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAutocompletion/shareAutocompletionSpecialChars.feature#L40)
```

Reply:
Interesting - I thought that @kiranparajuli589 had enhanced the expected-failures-file-linter to find formatting problems like this.